.babelrc

```diff
@@ -1,4 +1,5 @@
 {
+  "__mplheader": "This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.",
```

Comment:
I don't think this is the proper way to do the MPL header. You should just use comments.

Reply:
Is the Babel RC not JSON?